### ISSUE TITLE
Can't locate URI.pm in @INC

### ISSUE THREAD

**Issue by eserte** [rawleyfowler/Mojo-CouchDB#1]
It seems that Mojo-CouchDB-0.4 is missing a prereq declaration for URI:
```
Can't locate URI.pm in @INC (you may need to install the URI module) (@INC entries checked: ...  .) at /home/cpansand/.cpan/build/2023070721/Mojo-CouchDB-0.4-0/blib/lib/Mojo/CouchDB.pm line 10.
BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2023070721/Mojo-CouchDB-0.4-0/blib/lib/Mojo/CouchDB.pm line 10.
Compilation failed in require at t/01-local-integration.t line 5.
BEGIN failed--compilation aborted at t/01-local-integration.t line 5.
t/01-local-integration.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
...
```

**Comment by rawleyfowler** [comment]
Whoops, thought that was a part of `Mojo` already, I'll add it and upload asap. Thanks!

**Comment by rawleyfowler** [comment]
Fixed with #2, see v0.5
